src/storage/helper.c

```diff
@@ -106,12 +106,12 @@ storageHelperStanzaInit(const bool stanzaRequired)
     // If the stanza is NULL and the storage has not already been initialized then initialize the stanza
     if (!storageHelper.stanzaInit)
     {
-        if (stanzaRequired && cfgOptionStr(cfgOptStanza) == NULL)
+        if (stanzaRequired && cfgOptionStrNull(cfgOptStanza) == NULL)
             THROW(AssertError, "stanza cannot be NULL for this storage object");
 
```

Comment:
two things: 
1) Not shown here, but on lines 382-389 there are instances where cfgOptionTest is called and then a ternary operator is used. I was just thinking that the options here are always valid by this point so instead of 
`cfgOptionTest(cfgOptRepoS3Token) ? cfgOptionStr(cfgOptRepoS3Token) : NULL`
could they be simply
`cfgOptionStrNull(cfgOptRepoS3Token)`

2) In the same statement `strEqZ(cfgOptionStr(cfgOptRepoS3UriStyle), STORAGE_S3_URI_STYLE_HOST) ? storageS3UriStyleHost : storageS3UriStylePath` -- is it possible that they would "override" the default with empty/null and therefore defaults to storageS3UriStylePath? Or are you wanting a hard error here?

Reply:
> on lines 382-389 there are instances

Yes, updated.

> Or are you wanting a hard error here?

This option can't be NULL if the repo-type=s3 so a hard error here is good.

Reply:
There may be more places we can remove cfgOptionTest(), but I wasn't trying too hard to find them in this commit.